src/common/ITEMS/index.ts

```diff
@@ -33,6 +34,7 @@ const ITEMS = {
     SHAMAN,
     WARLOCK,
     WARRIOR,
+    CLASSIC
```

Comment:
I didn't bother breaking the classic item list into its own module like we did for the spell list. There isn't any overlap in ID's for items that I know of, so I didn't see any value of doing so for the classic items. There may be an argument for optimization, but we aren't adding very many items in classic for now.